topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -263,6 +265,39 @@ We have now decided that our input reads have to be filtered to an even higher s
 
 You can re-run a tool many times with different settings. Each time you re-run the tool, its new output datasets will appear at the top of your current history.
 
+
+## Convert your analysis history into a workflow
+
+When you look carefully at your history, you can see that it contains all steps of our analysis, from the beginning to the end. By building this history we have actually built a complete record of our analysis with Galaxy preserving all parameter settings applied at every step. But when you need to analyze new data, it would be tedious to do each step over again. Wouldn't it be nice to just convert this history into a workflow that we will be able to execute again and again?
+
+Galaxy makes this very easy with the `Extract workflow` option. This means any time you want to build a workflow, you can just perform the steps once manually, and then convert it to a workflow, so that next time it will be a lot less work to do the same analysis.
+
+> ### {% icon hands_on %} Hands-on: Extract workflow
```

Comment:
This is a much better version of `faqs/galaxy/workflows_extract_from_history.md`, would you want to expand the FAQ entry with your extra details + screenshots, and then use the FAQ here? `{% snippet faqs/galaxy/workflows_extract_from_history.md %}`

Reply:
@hexylena I've expanded the snippet and synced the other intro tutorials that teach how to extract a workflow from history, but didn't use the snippet in these because each tend to include extra images or text.

I'll leave to you if you would like to go ahead and use the snippet there (maybe with some placeholder) in a follow-up commit/PR.

Reply:
Fair enough! Thanks for doing that!